packages/docgen/src/index.js

```diff
@@ -126,14 +77,14 @@ module.exports = function( sourceFile, options ) {
 	} );
 
 	// Ouput
-	if ( result === undefined ) {
-		process.stdout.write(
-			'\nFile was processed, but contained no ES6 module exports:'
-		);
-		process.stdout.write( `\n${ sourceFile }` );
-		process.stdout.write( '\n\n' );
-		process.exit( 0 );
-	}
+	// if ( result === undefined ) {
+	// 	process.stdout.write(
+	// 		'\nFile was processed, but contained no ES6 module exports:'
+	// 	);
+	// 	process.stdout.write( `\n${ sourceFilePath }` );
+	// 	process.stdout.write( '\n\n' );
+	// 	process.exit( 0 );
+	// }
```

Comment:
Remove or restore this commented out section?

Reply:
Revived it.